pom.xml

```diff
@@ -135,6 +135,15 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- For Magic Number-->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.8</version>
+        </dependency>
+
+
+
```

Comment:
Read about spring-boot-starter parent dependency. It's not a spring boot project, but it will be useful for you and for the future